src/interface/meta.thrift

```diff
@@ -37,8 +38,7 @@ enum ErrorCode {
     E_UNKNOWN        = -99,
 } (cpp.enum_strict)
 
-
-enum AlterSchemaOp {
+enum AlterSchemaOptionType {
```

Comment:
Do we still need the changes?

Reply:
I think this way is clearer.
We only modify the `column` in `schem` here,  but  cannot modify the `property`in `schema`.

Reply:
For the naming,  we'd better leave it in another pr.  We could discuss it there.

Reply:
not only naming.

I think there are two reasons for this change:
1) In parser and graph, the option that modify column in the schema are uniformly defined as `AlterSchemaOption`, i.e.in  MaintainSentences.h

2) Before this feature, because there is only column in the schema, `AlterSchemaOp` modifies  the schema
This feature starts to add `property` in the schema, so the original `AlterSchemaOp` only modify the schema, it is not suitable (only modify the columns in the schema, can not modify the properties in the schema).

So modified here

Reply:
AlterSchemaOptionType and AlterSchemaOp seems to be same....

Reply:
`AlterSchemaOp` should be a shorthand for `AlterSchemaOperation`

Reply:
I will correct it according to your opinion here.